web_timeline/static/lib/vis/vis.css

```diff
@@ -0,0 +1,810 @@
+.vis .overlay {
```

Comment:
With npm you can install a concrete version, just like with `pip`. It would make the module much thinner, but nevermind. It's not a blocker.